src/helpers/validators-sort.ts

```diff
@@ -1,25 +1,84 @@
 import {ValidatorItem} from '@app/types';
 
+export enum ValidatorSortKey {
+  random, // default
+  name, // description.moniker
+  power, // power || tokens
+  commission, // localStatus
+}
+
 /**
  * @name validatorsSort
+ * @description Sorting validators list based on key param
+ * @param validators
+ * @param key
+ */
+export function validatorsSort(
+  validators: ValidatorItem[],
+  key: ValidatorSortKey,
+) {
+  switch (key) {
+    case ValidatorSortKey.random:
+      return validatorsSortRandom(validators);
+    case ValidatorSortKey.name:
+      return validatorsSortName(validators);
+    case ValidatorSortKey.commission:
+      return validatorsSortCommission(validators);
+    case ValidatorSortKey.power:
+      return validatorsSortPower(validators);
+    default:
+      return validatorsSortPower(validators);
```

Comment:
According to https://github.com/haqq-network/haqq-wallet/pull/1445/files#diff-4b21b0dba3374d039d6ef1b9af7d173e1aad524aae9e441aef848c6633bad7c3R50
```suggestion
      return validatorsSortRandom(validators);
```